tests/Set/DoctrineORM29Set/Fixture/cache.php.inc

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Rector\Doctrine\Tests\Set\DoctrineORM29Set\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * An Entity
+ *
+ * @ORM\Table(name="entity")
+ * @ORM\Cache(usage="READ_ONLY", region="entity_cache")
+ */
+class AnEntity
+{
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Doctrine\Tests\Set\DoctrineORM29Set\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * An Entity
+ */
+#[\Doctrine\ORM\Mapping\Table(name: 'entity')]
+#[\Doctrine\ORM\Mapping\Cache(usage: 'READ_ONLY', region: 'entity_cache')]
```

Comment:
Thanks for your works! :heart: 

Just a minor comment, since we already import `Doctrine\ORM\Mapping` as `ORM`, can't we use the `ORM` alias like this?
```suggestion
#[ORM\Table(name: 'entity')]
#[ORM\Cache(usage: 'READ_ONLY', region: 'entity_cache')]
```

Reply:
You can set auto import like the following config in your root rector.php : 

https://github.com/rectorphp/rector-src/blob/e76200b18a3792ea3266bb9f685b811d4049f210/rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/config/auto_import.php#L12

and then the result will use auto-import:

https://github.com/rectorphp/rector-src/blob/e76200b18a3792ea3266bb9f685b811d4049f210/rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/FixtureAutoImported/donot_reorder_annotation.php.inc#L60-L62

follow the alias of namespace/class seems a nice feature to have, feel free to create PR to [AnnotationToAttributeRector.php](https://github.com/rectorphp/rector-src/blob/main/rules/Php80/Rector/Class_/AnnotationToAttributeRector.php)